util/tracer_replayer.h

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
```

Comment:
I had the same thought but since it is not a big file. I follow the way of `file_reader_writer.h` :)